src/query_builder.js

```diff
@@ -163,6 +170,46 @@ function containsWildcard(str) {
   return wildcardRegex.test(str);
 }
 
+function containsIndex(str) {
+    var wildcardRegex = /\$/;
```

Comment:
What if alias contains $ sign without, but there is not an index?
```
current amount, $ per day
```
I think, better pattern is `/\$\d/g`. What you think?